automation/release.sh

```diff
@@ -110,6 +112,7 @@ function main() {
 
     build_release_artifacts
     update_github_release
+    update_github_source_tarball_signature
```

Comment:
I was expecting to see modification to update_github_release as well. There is no reason to push release and then sign it. Imho it is counter intuitive in this case.

Reply:
Thanks